containers/index.tsx

```diff
@@ -11,15 +11,15 @@ type WithAppContainersProps = {
 };
 
 export const WithAppContainers: FC<WithAppContainersProps> = ({ children }) => (
-	<Connector.Provider>
-		<BlockExplorer.Provider>
-			<TransactionNotifier.Provider>
-				<Loans.Provider>
-					<SideNav.Provider>{children}</SideNav.Provider>
-				</Loans.Provider>
-			</TransactionNotifier.Provider>
-		</BlockExplorer.Provider>
-	</Connector.Provider>
+	<SideNav.Provider>
```

Comment:
Where does this change come from?

Reply:
its because the way its structured, the network error message is rendered by the SideNav.Provider and since most of the logic controlling that is connector it needs to be rearranged